src/org/labkey/test/components/ui/grids/ManageGridViewsDialog.java

```diff
@@ -0,0 +1,174 @@
+package org.labkey.test.components.ui.grids;
+
+import org.junit.Assert;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Dialog used to manage saved views.
+ */
+public class ManageGridViewsDialog extends ModalDialog
```

Comment:
The name should match the product component.
```suggestion
/**
 * Dialog used to manage saved views. Wraps ManageViewsModal.tsx in UI components.
 */
public class ManageViewsDialog extends ModalDialog
```

Reply:
Updated.